ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java

```diff
@@ -58,4 +59,10 @@ public void checkHomePageVisibility(String username) {
 		SyndesisRootPage syndesisRootPage = new SyndesisRootPage();
 		syndesisRootPage.getRootElement().shouldBe(Condition.visible);
 	}
+
+	@Then("^(\\w+)? is presented with the Syndesis page \"([^\"]*)\"$/")
```

Comment:
Pls merge it with method `checkHomePageVisibility` and the regex doesn't seem right to me :)